src/server/common/helpers/redis-client.js

```diff
@@ -0,0 +1,65 @@
+import { Redis } from 'ioredis'
+
+import { createLogger } from './logging/logger.js'
+
+import config from '~/src/server/config.js'
+
+/**
+ * Setup Redis and provide a redis client
+ *
+ * Local development - 1 Redis instance
+ * Out in the wild - Elasticache / Redis Cluster with username and password
+ * @returns {import('ioredis').Cluster | import('ioredis').Redis}
```

Comment:
This one's inferred automatically (if you hover over `buildRedisClient()`)

```suggestion
```

Reply:
Cheers. Done.